src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+
+  &:last-of-type {
+    padding-bottom: 0;
+  }
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &--1 :first-child,
+  &--2 :nth-child(-n + 2),
+  &--3 :nth-child(-n + 3),
+  &--4 :nth-child(-n + 4),
+  &--5 :nth-child(-n + 5) {
```

Comment:
The selectors for the star states (e.g., `&--1 :first-child`, `&--2 :nth-child(-n + 2)`) might not be specific enough to target the intended elements. Ensure that these selectors correctly apply the active star image to the right elements. Consider using more specific selectors if necessary.